sources/engine/Xenko.Graphics/TextureOptions.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+using System;
+
+namespace Xenko.Graphics
+{
+    /// <summary>
+    /// Resource options for textures.
+    /// </summary>
+    /// <remarks>
+    /// This enumeration is used in TextureDescription.The TextureOptions
+    ///     must be 'None' when creating textures with CPU access flags.   
+    /// </remarks>
+    [Flags]
+    public enum TextureOptions
+    {
+        /// <summary>
+        /// None. The default.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Enables resource data sharing between two or more Direct3D devices. The only
```

Comment:
I think it is better to keep a single line in the` <summary>` and put everything else in `<remarks>`.